cockroachcloud/export-logs.md

```diff
@@ -173,37 +170,37 @@ Perform the following steps to enable log export from your {{ site.data.products
             {% include_cached copy-clipboard.html %}
             ~~~json
             {
-             type: AWS_CLOUDWATCH,
-             log_name: "default",
-             auth_principal: "{role_arn}",
-             redact: true,
-             region: "",
-             groups: [
+             "type": "AWS_CLOUDWATCH",
+             "log_name": "crdb-default",
+             "auth_principal": "{role_arn}",
+             "redact": true,
+             "region": "",
+             "groups": [
                      {
-                         log_name: "sql",
-                         channels: ["SQL_SCHEMA", "SQL_EXEC"],
-                         redact: false,
+                         "log_name": "crdb-sql",
+                         "channels": ["SQL_SCHEMA", "SQL_EXEC"],
+                         "redact": false,
                      },
                      {
-                         log_name: "devops",
-                         channels: ["OPS", "HEALTH", "STORAGE"]
-                         min_level: "WARNING"
+                         "log_name": "devops-sql",
```

Comment:
nit: can you use `crdb-devops` instead of `devops-sql`?

Reply:
You [asked](https://github.com/cockroachdb/docs/pull/16569#discussion_r1145424687) for `devops-sql`. Sure, changed it 8x.